### DISCUSSION TITLE
Enterprise Multiserver fails at AppASG and StreamASG upon CloudFormation Update

### DISCUSSION THREAD

**Discussion by ikenichiro** [mozilla/hubs#6064] [Hubs Cloud Questions]
**Description**
Updating an Enterprise Multiserver stack fails for AppASG and StreamASG because of timeout.
It is partially reported in #6002 and #6004 as also, but those issue also include a different issue so I'm reporting a new issue specific to this problem,

**To Reproduce**
1. Setup a working v1.1.5 stack (My Region is at Asia Pacific, ap-northeast-1), check everything is working, and then update the stack to offline mode from CloudFormation stack update function.
2. Make the CloudFormation stack to online, have the number of app servers and number of voice/videos streaming servers with the value more than 1.
3. Update the stack and wait until the resource creation,
4. AppASG and StreamASG stops at UPDATE_IN_PROGRESS after creating new instance and waiting for signal(s), wait for 30 minutes and see error.

**Expected behavior**
Have the CloudFormation stack to complete the update and show UPDATE_COMPLETE, accessible hubs cloud running on AWS.

**Screenshots**
<img width="855" alt="appasg-streamasg-error" src="https://user-images.githubusercontent.com/3268073/227069417-6c5d55ee-7b8c-4efc-8cd9-ba7df0cec127.png">

**Hardware**
 - Device: Desktop
 - OS: Windows 10 
 - Browser: Firefox 111.0.0, 111.0.1

**Additional context**
- AWS Region: ap-northeast-1 (Tokyo)
- Hubs Cloud: No customization, including no custom client.
- The values for number of servers seems not to affect anything, I've tried up to 1 to 3 servers but it always fails with 0 SUCCESS

***Things I have tried but did not work***
- Try only with application server (0 for stream)
- Change EC2 instance type (c5.large, c5.xlarge)
- Change Load Balancing Method to DNS  Round Robin

**Comment by NigelRidderhof** [comment]
I'm also running into these issues. Don't have much to bring into the conversation besides that it seems to me that multiple people are running into this around the same time. Maybe that could suggest there's something out of our reach that has changed? Not sure, just throwing it out there. 

Personally hoping the Hubs team can reproduce it soon.

**Comment by eyesnareinc** [comment]
OK, it happened again with a NEW Stack. I just tried to update a new stack (moving it from offline to online). The stack updated seemed to stall in UPDATE_IN_PROGRESS and eventually the updated failed at AppASG.
![HubsCloudStall_FAIL](https://user-images.githubusercontent.com/66790476/227237087-3cc03017-9fd5-4f71-97a4-696c7cb4e242.JPG)

**Comment by eyesnareinc** [comment]
Stack update failed (moving the stack from offline to online) for the brand new backup stack. I would love to find out why this is happening. 
![HubsCloudStall_FAIL2](https://user-images.githubusercontent.com/66790476/227252538-1550f5ac-c12c-4158-b59e-0fe22451f992.JPG)

**Comment by pattersonbl2** [comment]
A workaround for this issue has been pinned to the top of the discussions page.

**Reply by FrankAllanRJ** [reply]
Where? I haven't found it yet.

Best regards

**Comment by FrankAllanRJ** [comment]
I still can't rewire my stack. 
`StreamASG | UPDATE_FAILED |  Received 0 SUCCESS signal(s) out of 1. Unable to satisfy 100% MinSuccessfulInstancesPercent requirement`
https://github.com/mozilla/hubs/discussions/6063#discussioncomment-6054077
